server/test/follow.spec.js

```diff
@@ -0,0 +1,82 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+chai.use(chaiHttp);
+let userToken;
+before(async () => {
+  const userResponse = await chai
+    .request(app)
+    .post('/api/v1/auth/login')
+    .send({
+      email: 'mofe@okoro.com',
+      password: 'soldier'
+    });
+
+  userToken = userResponse.body.data.token;
+});
+after(async () => {
+
+});
+describe('User can follow and unfollow a user route', () => {
+  it('should follow a user', async () => {
+    const res = await chai.request(app)
+      .post('/api/v1/profiles/Dimkpa/follow')
+      .set({ Authorization: `Bearer ${userToken}` });
+    expect(res).to.have.status(200);
+    expect(res.body.message).to.equal('Successfully followed user');
+    expect(res.body.data).to.have.property('personToBeFollowed');
+    expect(res.body.data).to.have.property('followers');
+  });
+  it('should not follow a user if the user doesnt exist', async () => {
+    const res = await chai.request(app)
+      .post('/api/v1/profiles/Dimkpayy/follow')
+      .set({ Authorization: `Bearer ${userToken}` });
+    expect(res).to.have.status(404);
+    expect(res.body.message).to.equal('You cannot follow a user that does not exist');
+  });
+  it('should not a follow a user if the user has been followed before', async () => {
```

Comment:
Good job @geekdanitek, kindly check this line for grammatical errors

Reply:
Ok, eagles eyes 👍